node/network/protocol/src/peer_set.rs

```diff
@@ -181,6 +174,262 @@ impl<T> IndexMut<PeerSet> for PerPeerSet<T> {
 ///
 /// Should be used during network configuration (added to [`NetworkConfiguration::extra_sets`])
 /// or shortly after startup to register the protocols with the network service.
-pub fn peer_sets_info(is_authority: IsAuthority) -> Vec<sc_network::config::NonDefaultSetConfig> {
-	PeerSet::iter().map(|s| s.get_info(is_authority)).collect()
+pub fn peer_sets_info(
+	is_authority: IsAuthority,
+	peerset_protocol_names: &PeerSetProtocolNames,
+) -> Vec<sc_network::config::NonDefaultSetConfig> {
+	PeerSet::iter()
+		.map(|s| s.get_info(is_authority, &peerset_protocol_names))
+		.collect()
+}
+
+/// Supported validation protocol versions. Only versions defined here must be used in the codebase.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, EnumIter)]
+pub enum ValidationVersion {
+	/// The first version.
+	V1 = 1,
+}
+
+/// Supported collation protocol versions. Only versions defined here must be used in the codebase.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, EnumIter)]
+pub enum CollationVersion {
+	/// The first version.
+	V1 = 1,
+}
```

Comment:
These are not used anywhere.

Reply:
The only place they are used is https://github.com/paritytech/polkadot/pull/5876/files#diff-64aa50a62fe0bddebd18c842650162a2b1b86b0abda390cc325658bb07d5e222L778-R840